language/stdlib/modules/LibraAccount.move

```diff
@@ -891,5 +891,25 @@ module LibraAccount {
     ///////////////////////////////////////////////////////////////////////////
     // End of the proof of concept code
     ///////////////////////////////////////////////////////////////////////////
+
+    // ****************** SPECIFICATIONS *******************
+
+    spec module {
+        /// Returns field `key_rotation_capability` of the
+        /// LibraAccount under `addr`.
+        define spec_get_key_rotation_cap(addr: address):
+            Option<KeyRotationCapability>
```

Comment:
Put on same line? Also consistent use of `{`. Next function has it on same line. I think this is the preferred style.